deploy/crds/apps.openshift.io_servicebindingrequests_crd.yaml

```diff
@@ -0,0 +1,210 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Why are we adding this file?

Reply:
That was autogenerated. I have removed it.8149e90d0df51eb0cbc93ed3aa58177915826f7e